behavioral-tests/run.sh

```diff
@@ -15,6 +15,11 @@ for cs in $(find . -mindepth 1 -maxdepth 1 -type d | sort)
 do
     (
     cd $cs
+    if test ! -e expected.log
```

Comment:
This skipping does not make me happy. I promise to take the time to consider how we can avoid it.